third_party/llvm/llvm.bzl

```diff
@@ -46,48 +46,6 @@ def gentbl(name, tblgen, td_file, td_srcs, tbl_outs, library = True, **kwargs):
     native.cc_library(name=name, textual_hdrs=[f for (_, f) in tbl_outs],
                       includes=includes,  **kwargs)
 
-
-# Rule for simple expansion of template files. This performs a simple
-# search over the template file for the keys in substitutions,
-# and replaces them with the corresponding values.
-#
-# Typical usage:
-#   load("/tools/build_rules/expand_header_template", "expand_header_template")
-#   expand_header_template(
-#       name = "ExpandMyTemplate",
-#       template = "my.template",
-#       out = "my.txt",
-#       substitutions = {
-#         "$VAR1": "foo",
-#         "$VAR2": "bar",
-#       }
-#   )
-#
-# Args:
-#   name: The name of the rule.
-#   template: The template file to expand
-#   out: The destination of the expanded file
-#   substitutions: A dictionary mapping strings to their substitutions
-
-def expand_header_template_impl(ctx):
-  ctx.template_action(
-      template = ctx.file.template,
-      output = ctx.outputs.out,
-      substitutions = ctx.attr.substitutions,
-  )
-
-expand_header_template = rule(
```

Comment:
not this specific file, but llvm.BUILD uses expand_header_template, so I added a load statement there.